src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitIconButtonTests.cs

```diff
@@ -23,14 +23,14 @@ public class BitIconButtonTests : BunitTestContext
                DataRow(Visual.Material, true, "Emoji2", "I'm Happy", true),
                DataRow(Visual.Material, false, "Emoji2", "I'm Happy", false),
            ]
-        public void BitIconButtonTest(Visual visual, bool isEnabled, string iconName, string toolTip, bool expectedResult)
+        public void BitIconButtonTest(Visual visual, bool isEnabled, string iconName, string title, bool expectedResult)
```

Comment:
please remove the "expectedResult" argument and calculate its desired value in the test method body based on the logic we expect to be applied.